simulators/main/simulators.py

```diff
@@ -484,45 +83,50 @@ def load(filename):
         return {}
 
 
-def datetime_to_string(date_time, time_string_format="%Y-%m-%dT%H:%M:%S.%f"):
-    return date_time.strftime(time_string_format)[:-3] + 'Z'
-
-
 ###################################################################################
 log.info(f'cwd:{os.getcwd()}')
-SIMULATOR_MODELS = load("simulators.json")
-
-simulators = list(map(lambda model: MachineSimulator(model), SIMULATOR_MODELS))
-
-# create managed object for every simulator
-[item.get_or_create_device_id() for item in simulators]
+DEVICE_MODELS = load("simulators.json")
+DEVICE_EVENT_MODELS = []
+DEVICE_MEASUREMENT_MODELS = []
+
+# Add device id to the model of devices
+for device_model in DEVICE_MODELS:
+    device_id = get_or_create_device_id(device_model)
```

Comment:
Please introduce self reference everywhere

Reply:
What do you mean by this?